src/storage/wal_replay.cpp

```diff
@@ -403,47 +404,86 @@ void ReplayState::ReplayDropTableMacro(BinaryDeserializer &deserializer) {
 // Replay Index
 //===--------------------------------------------------------------------===//
 void ReplayState::ReplayCreateIndex(BinaryDeserializer &deserializer) {
-	auto info = deserializer.ReadProperty<unique_ptr<CreateInfo>>(101, "index");
+
+	auto create_info = deserializer.ReadProperty<unique_ptr<CreateInfo>>(101, "index_catalog_entry");
+	auto index_info = deserializer.ReadProperty<IndexStorageInfo>(102, "index_storage_info");
```

Comment:
Ideally we would avoid calling `ReadProperty` explicitly here and we would move this into a struct whose serialization methods can be generated through the serialize script